lib/screens/v2/screen_data/parameters.ex

```diff
@@ -66,9 +66,24 @@ defmodule Screens.V2.ScreenData.Parameters do
     end
   end
 
-  @spec audio_interval_minutes(Screen.app_id()) :: pos_integer() | nil
-  @spec audio_interval_minutes(Screen.app_id(), static_params()) :: pos_integer() | nil
-  def audio_interval_minutes(app_id, static_params \\ @static_params) do
+  @spec audio_interval_minutes(Screen.t(), static_params()) :: pos_integer() | nil
+  def audio_interval_minutes(screen, static_params \\ @static_params)
+
+  def audio_interval_minutes(
+        %Screen{
+          app_id: app_id,
+          app_params: %{audio: %ScreensConfig.V2.Audio{interval_enabled: interval_enabled}}
+        },
+        static_params
+      ) do
+    if interval_enabled, do: fetch_interval_minutes(app_id, static_params), else: 0
```

Comment:
By the existing typespec this function is allowed to return `nil`, so let's return `nil` in this case as well rather than using `0` to be clear that the intent is to have no interval. (I mentioned using 0 earlier since the client-side code does also disable readouts when the interval is 0, but I see I misremembered how we handled this on the server.)